### PR TITLE
Aswathy/account type cellxpert v2

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/form/wizard/index.tsx [hunk 1]
```diff
@@ -35,6 +36,15 @@ const Modal = styled.div`
     max-width: 1200px;
     z-index: 100;
     border-radius: 8px;
+    @media (max-width: 768px) {
```
Comment (prince-deriv) [inline src/components/form/wizard/index.tsx:39]: Can we use the one we have on `devices` ?

File: src/components/form/wizard/stepper.tsx [hunk 2]
```diff
@@ -8,6 +8,9 @@ const StepperWrapper = styled.div`
     justify-content: space-between;
     width: 700px;
     margin: 0 auto;
+    @media (max-width: 768px) {
```
Comment (prince-deriv) [inline src/components/form/wizard/stepper.tsx:11]: here as well.

File: src/pages/signup-affiliates/components/_account-type.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import { Header } from 'components/elements'
+
+const AccountType = () => {
+    const MainWrapper = styled.div`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    `
+    const card: CardProps[] = [
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/components/_account-type.tsx:16]: it's a collection and thus the variable name should be plural.

File: src/pages/signup-affiliates/wizard.tsx [hunk 4]
```diff
@@ -1,11 +1,28 @@
...
+import { WithIntl, localize } from 'components/localization'
+import { Wizard } from 'components/form'
 
 const AffiliateSignup = () => {
+    const Steps = [
+        'Account type',
+        'Address details',
+        'Phone number',
+        'Personal details',
+        'terms of use',
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/wizard.tsx:13]:
```suggestion
        'Terms of use',
```

File: src/components/form/wizard/index.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/components/form/wizard/index.tsx:39]: okay

File: src/components/form/wizard/stepper.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/components/form/wizard/stepper.tsx:11]: okay

File: src/pages/signup-affiliates/components/_account-type.tsx (hunk 3 above)
Comment (aswathy-deriv) [reply src/pages/signup-affiliates/components/_account-type.tsx:16]: ok

File: src/pages/signup-affiliates/wizard.tsx (hunk 4 above)
Comment (aswathy-deriv) [reply src/pages/signup-affiliates/wizard.tsx:13]: okay

File: src/components/custom/_card.tsx [hunk 5]
```diff
@@ -21,6 +21,9 @@ const StyledCard = styled.div<{ selected: boolean }>`
     border: ${(props) => (props.selected ? '1px solid #aad2d8' : '1px solid #ffffff')};
     border-radius: 8px;
     cursor: pointer;
+    @media (max-width: 768px) {
```
Comment (prince-deriv) [inline src/components/custom/_card.tsx:24]: Here as well

Comment (aswathy-deriv) [reply src/components/custom/_card.tsx]: ok

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**